core/eth/directory/contract_names.go

```diff
@@ -4,12 +4,18 @@ package directory
 // It is ok to remove contracts from this list if the offchain code doesn't interact with them anymore.
 
 const (
-	ServiceManager         ContractName = "SERVICE_MANAGER"
-	EigenDAEjectionManager ContractName = "EIGEN_DA_EJECTION_MANAGER"
```

Comment:
@cody-littley this was a type, the contract is actually registered under `EJECTION_MANAGER` only